language-support/ts/daml-react/README.md

```diff
@@ -47,23 +47,16 @@ Now you can use the following React hooks to interact with a DAML ledger:
 const party = useParty()
 ```
 
-`useExercise`
+`useLedger`
 -------------
-`useExercise` returns a function to exercise a choice by contract id.
+`useLedger` returns an instance of the `Ledger` class of @daml/ledger to interact with the DAML
```

Comment:
Can we please make the reference to `@daml/ledger` a link?

Reply:
done.